packages/schematics/angular/library/index.ts

```diff
@@ -147,6 +147,11 @@ function addAppToWorkspaceFile(
             tsConfig: `${projectRoot}/tsconfig.lib.json`,
             project: `${projectRoot}/ng-package.json`,
           },
+          configurations: {
+            production: {
+              tsConfig: `${projectRoot}/tsconfig.lib.prod.json`,
```

Comment:
We also need a migration for this, right?

Reply:
There is already a separate PR for that, I made it separate since it can go in now in master.
https://github.com/angular/angular-cli/pull/15272

I also, created some utils, which would be needed for another schematic